### ISSUE TITLE
Window display with new stats panel.

### ISSUE THREAD

**Issue by grobson13** [HearthSim/Hearthstone-Deck-Tracker#3273]
## Bug report
Window is cutoff when I go full screen since new stat panel was added.

### Expected Behavior
Expect it show the whole screen.

### Actual Behavior
Describe the actual behavior here

### Steps to reproduce behavior
I just have to maximize the window. My secondary screen isn't big but should be able to resize to accomadate this. I would much rather have a small deck list panel than having the right side cut off. Resizable panels would be great.

### Log/Screenshots
If applicable, provide a log of the session where the problem occoured.  
See here: https://github.com/HearthSim/Hearthstone-Deck-Tracker/wiki/Reporting-Issues

![heathstone](https://cloud.githubusercontent.com/assets/14284969/26756347/f3e793b8-4865-11e7-9d5e-bfdef3a6f2db.png)

**Comment by azeier** [comment]
Yeah :/ I'm not sure what to make reasonably smaller without causing overlapping here.
Since 1.3.5 you can disable the my games panel under `options (advanced) > tracker > general`. But that's far from an actual solution, too.

Making the deck panel smaller also only helps so far, I'm open to suggestions here.

**Comment by grobson13** [comment]
For me it would be the left panel, look at the screenshot there is so much blank space. Maybe making the classes and the type of deck into drop down menus would allow that panel to be a lot less wide. Also could that same panel be resizable. Those same menu's could wrap down onto next line as it is made smaller. The same menus could also be moved up into the menu bar so the panel could be made smaller. Are you referring to the deck panel as the left or right? The list of cards is pretty important so I would rather the left panel was made smaller. Maybe these ideas will help. Thanks

**Comment by grobson13** [comment]
Maybe in the mean time you could move the exit button over so it could be viewed.
